lib/models.js

```diff
@@ -62,6 +62,10 @@ class DataElementSpecifications {
     return this.all.filter(de => de.isEntry);
   }
 
+  get abstracts() {
+    return this.all.filter(de => de.isAbstract);
+  }
```

Comment:
Is there a use case for when we would want to retrieve all abstracts?  If so, then great!  If not, then I'd suggest we don't define this function -- because it's just more lines of code to maintain with no added value.

There very well may be a use case.  I'm just asking to confirm.